models.py

```diff
@@ -3,38 +3,44 @@
 
 db = SqliteExtDatabase('elo.db', pragmas=(('foreign_keys', True),))
 
+needs_table_li = []
```

Comment:
??

Reply:
List of classes that need a table generated. It was originally called `needs_table`, but that's what I named the decorator.